torch_ttnn/passes/lowering/add_data_move_pass.py

```diff
@@ -83,6 +83,16 @@ def is_tt_data_move(node) -> bool:
     ]
 
 
+# For non-tiled shapes, these ttnn ops require layout changes
+layout_change_ops = set(
```

Comment:
Alright, so we need to query if an op supports non-tilized tensors so that this can be done in a better way?